projects/js-packages/components/package.json

```diff
@@ -19,11 +19,19 @@
 		"@automattic/jetpack-base-styles": "workspace:^0.1.10",
 		"@babel/core": "7.16.0",
 		"@babel/preset-react": "7.16.0",
+		"@types/enzyme": "3.10.11",
+		"@types/jest": "27.4.1",
+		"@types/react": "17.0.40",
+		"@types/react-dom": "17.0.13",
+		"@types/react-test-renderer": "17.0.1",
+		"chai": "4.3.4",
+		"enzyme": "3.11.0",
```

Comment:
I note formerly these were being loaded via js-test-runner. Does typescript mean we can't do that anymore?

Reply:
We can do that but then we need to add the corresponding `@types` package if it's available separately, otherwise TS won't be happy.